TemplateManagement/PowerShell/ActiveDirectoryJoinV2/readme.md

```diff
@@ -13,14 +13,14 @@ The steps in this section show how to set up the sample scripts that uniquely re
 ### Prerequisites
 - **Lab creation**: The lab must be created with the **Create a template virtual machine** option enabled.  If you are planning students to AAD join their lab VMs, you should also enable the **Use same password for all virtual machines** option.
 - **Template VM setup**: You should install all other software/customizations *before* you set up the scripts to rename.  Also, we *don't* recommend exporting the template VM image with the scripts set up because they may cause ill-side effects (e.g., script inadvertently run on startup, errorneous values saved in the SecretStore, etc.)
-- **Windows versions**: The template and student VMs should use a Windows 10/11 image.
+- **Windows versions**: The template and student VMs should use a Windows 10/11 image.  This has not been tested on Windows server versions.
```

Comment:
There are really 2 scenarios here (as I'm reading through) which require different scripts.  Perhaps those each go in subdirectories for the scenario?  That way you could have a high-level readme and then just point to the subdirectories for each of the two scenarios.  It also might be interesting to use an aka.ms link to point at the sub directories to see how many people pick each path?